synapse/storage/databases/main/roommember.py

```diff
@@ -666,21 +679,24 @@ async def get_joined_users_from_context(
             state_group = object()
```

Comment:
A place where we could us https://peps.python.org/pep-0661/ in the future... if it's accepted